SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/ExpectedGeneratedSchemaFileContents.h

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+namespace ExpectedFileContent
+{
+
+const char ASpatialTypeActor[] = "\
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved\r\n\
```

Comment:
When we need to update our 'golden schema' here doesn't that mean the person needs to also manually add these line endings and such?
Perhaps we should just include `goldenschema.schema` and then read the files, makes updating easier.

Reply:
Do you suggest moving the contents of this file to corresponding _golden.schema files?

Reply:
Yeah pretty much

Reply:
Or use a string type where all of the `\r\n\` are interpreted

Reply:
Give the added complexity of reading in reference files and how infrequently we change schema format I think having them in this file is fine for now.